server/index.js

```diff
@@ -1,14 +1,20 @@
 const express = require('express');
-var bodyParser = require('body-parser');
+
 let app = express();
-app.use(express.static(__dirname + '/../client/dist'));
+var bodyParser = require('body-parser');
+let retrieveOne = require('../database/seed.js').retrieveOne;
 
-app.post('/', function (req, res) {
-  res.status(201).send();
-});
+app.use(bodyParser.json());
+app.use(express.static(__dirname + '/../client/dist'));
 
-app.get('/', function (req, res) {
-  res.status(200).send('testing');
+app.get('/shop/:productId', (req, res) => {
+  let item = req.params.productId;
+  retrieveOne(item, (err, data) => {
+    if (err) {
+      res.status(404).send('error');
+    }
+    res.status(200).send(data);
```

Comment:
add a `.end()` after you send the data, otherwise the GET request remains open indefinitely and the browser will eventually throw an error after timing out.